jest-coverage/coverage-final.json

```diff
@@ -0,0 +1,28 @@
+{"/home/iamnwani/Desktop/OSS/ambianic-ui/src/App.vue": {"path":"/home/iamnwani/Desktop/OSS/ambianic-ui/src/App.vue","statementMap":{"0":{"start":{"line":19,"column":0},"end":{"line":19,"column":null}}},"fnMap":{},"branchMap":{},"s":{"0":1},"f":{},"b":{}}
```

Comment:
@vickywane this coverage dump should be in .gitignore

Reply:
I would add it.

Reply:
I have added the `jest-coverage` directory to the `.gitignore`

Reply:
@vickywane why is this file still in the PR?